examples/stage4_template.spec

```diff
@@ -171,15 +171,37 @@ stage4/root_overlay:
 # stage4/xinitrc:
 stage4/xinitrc:
 
-# This option is used to create non-root users on your CD.  It takes a space
-# separated list of user names.  These users will be added to the following
-# groups: users,wheel,audio,games,cdrom,usb
-# If this is specified in your spec file, then the first user is also the user
-# used to start X. Since this is not used on the release media, it is blank.
-# example:
+# This option is used to create groups. It takes a carriage-return separated
+# list of group names. For instance:
+# stage4/groups:
+#     admin
+#     web_group
+#     sudo_group
+stage4/groups
```

Comment:
Missing a `:` on the end of this line. Fixed locally.

Reply:
fixed too